app/controllers/devise_token_auth/passwords_controller.rb

```diff
@@ -181,7 +181,11 @@ def password_resource_params
     end
 
     def render_not_found_error
-      render_error(404, I18n.t('devise_token_auth.passwords.user_not_found', email: @email))
+      if Devise.paranoid
+        render_error(404, I18n.t('devise_token_auth.passwords.sended_paranoid'))
```

Comment:
Just as you've updated the response message to indicate success regardless of whether or not it worked, I think this should respond with a 200 to be consistent with the success response.